tests/lib/ignored-paths.js

```diff
@@ -164,6 +164,20 @@ describe("IgnoredPaths", () => {
                 )
             );
         });
+
+        it("should load package.json's eslintIgnore files if no specified ignore file", () => {
+            const ignoredPaths = new IgnoredPaths({ ignore: true, cwd: getFixturePath("package-json-ignore") });
+
+            assert.isTrue(ignoredPaths.contains("hello.js"));
+            assert.isTrue(ignoredPaths.contains("world.js"));
+        });
+
+        it("should not load package.json's eslintIgnore files if specified ignore file", () => {
+            const ignoredPaths = new IgnoredPaths({ ignore: true, cwd: getFixturePath() });
+
+            assert.isFalse(ignoredPaths.contains("hello.js"));
+            assert.isFalse(ignoredPaths.contains("world.js"));
```

Comment:
Can you add an assertion that the .eslintignore file does get used in this case?

```js
assert.isTrue(ignoredPaths.contains("sampleignorepattern"));
```